workspace/apps/pidp/src/app/features/auth/pages/link-account-confirm/link-account-confirm-resource.service.ts

```diff
@@ -40,6 +42,8 @@ export class LinkAccountConfirmResource {
   }
 
   public cancelLink(): Observable<Observable<void> | boolean> {
+    this.cookieService.delete('UserName');
```

Comment:
can we write a service called cookie.service which has function like 
delete(key: string) {
}
set(key:string, value: string) {
}
get(key:string) {
}
So that In future we can maintain easily by adding the functions to cookie.service and we can re-use them in different places. Then cookie service logic will be completely decoupled from the components.